### DISCUSSION TITLE
Serverless express app showing connection timed out when I try to accesss

### DISCUSSION THREAD

**Discussion by spicybirsge** [vercel/community#7148] [Help]
### Summary

When I try visiting my website in my browser I cannot access the link its an express app. The issue seems to be only for this deployment on vercel as I can access other vercel deployments. And also my nextjs app which has ssr can access the URL. But when I visit it I can connection timed out and I have no logs printed.
![image](https://github.com/vercel/community/assets/92243459/14ce4f0b-73ce-4673-aa0b-dda955cf2093)
I also don't think this is an issue with my code since no request is printed in the logs either. 

### Example

https://backend-mdshare.vercel.app/

### Steps to Reproduce

https://github.com/spicybirsge/markdown-share

**Comment by spicybirsge** [comment]
I would like to also add that it sometimes works and sometimes it doesn't (mostly it doesn't)

**Reply by QuiiBz** [reply]
Hi, this doesn't look like an issue with your code nor with Vercel. If your function times out, you'll see a Vercel timeout error page instead of your browser's `ERR_CONNECTION_TIMED_OUT` page.

Could you try to access it via another browser, device, and internet connection? I don't have any issues to access the linked URL on my side.

**Reply by spicybirsge** [reply]
This issue seems to be resolved for now however I would like to add that the url is sometimes extremely slow to load and sometimes shows connection timed out error(before I couldn't even access it), but when I use mobile data and visit it there is no issues, maybe it could be an issue with my WiFi but its wierd since all other websites work fine and a lot of vercel hosted projects including the vercel dashboard has been really slow when using WiFi, it didn't used to be so slow and idk why it only affects vercel website.

**Reply by QuiiBz** [reply]
That could be an issue with your ISP or default DNS resolver, since we don't have any other reports like this. I'd recommend to contact your ISP and ask them if there is any ongoing issue connecting to `76.76.21.0/24` - and to switch your default DNS resolver to Google, Cloudflare, dns0, etc.
